src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(object, number) {
```

Comment:
The 'add', 'subtract', 'multiply' and 'divide' methods should not take 'object' as a parameter. They should operate on the 'result' property of the 'calculator' object directly.